### DISCUSSION TITLE
AsyncClient, app parameter wiping

### DISCUSSION THREAD

**Discussion by RKN01011** [encode/httpx#3449] [General]
`@pytest.fixture(scope="session")
async def test_client() -> typing.AsyncGenerator[AsyncClient, None]:
    async with AsyncClient(app=application, base_url="http://test", timeout=600) as test_client:
        yield test_client`
        
We had an AsyncClient with the app parameter, but now it seems to have disappeared in the new version. Why is that? I haven’t found any information about this in the documentation

**Comment by lovelydinosaur** [comment]
You'll need to take a look at the CHANGELOG for this.

**Comment by dingus9** [comment]
For those coming after:
```
Deprecated
The app=... shortcut has been deprecated. Use the explicit style of transport=httpx.WSGITransport() or transport=httpx.ASGITransport() instead.
```
